### ISSUE TITLE
What is meant by repeat for Task2

### ISSUE THREAD

**Issue by anshumitts** [vineetm/ell-881-2018-deep-learning#26]
Do we have to use word embeddings and encoder RNN from checkpoint of task1?

**Comment by vineetm** [comment]
@anshumitts Yes. You are sharing encoder RNN across all three training tasks.

**Comment by prateek27** [comment]
@vineetm Sharing RNN means the weights will be trained on task1,  then those weights will be fed to task 2 (say NMT), then those weights will be adjusted again during NMT training and so for task 3. This is what I am assuming.

**Comment by vineetm** [comment]
@prateek27 Yes! That is exactly right.
